messenger/messenger.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 const (
-	sendAPIURL    = "https://graph.facebook.com/v2.6/me/messages?access_token="
+	sendAPIURL    = "https://graph.facebook.com/v9.0/me/messages?access_token="
 	profileURL    = "https://graph.facebook.com/v2.6/me/messenger_profile?access_token="
```

Comment:
do we need to update profile api to v9 as well?

Reply:
I'm modify profile API in another PR